launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -78,9 +68,9 @@ def __init__(
         super().__init__(**kwargs)
         self._policy = policy
         self._node_descriptions = node_descriptions
-        self._completed_future = None
-        self._started_task = None
-        self._container = None
+        self._completed_future = None  # type: Optional[Future]
```

Comment:
nice - didn't know about this syntax for mypy

Reply:
:+1: